assets/activation-scripts/activate

```diff
@@ -177,6 +194,12 @@ else
 
 fi
 
+# Start services before the shell or command is invoked
```

Comment:
(nb) I wondered whether it would be nice to organise this in the `profile.d` directory alongside CUDA but it doesn't match the description of "flox environment customizations" and we need this to happen after the user hooks and checks for re-activation.